Resources/Prototypes/_CM14/Entities/Structures/Doors/Windoors/assembly.yml

```diff
@@ -62,9 +67,14 @@
         acts: ["Destruction"]
       - !type:SpawnEntitiesBehavior
         spawn:
-          SheetPlasteel1: #todo cm14 look up
+          CMShardGlass:
+            min: 1
+            max: 1
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          CableApcStack1:
             min: 1
-            max: 2
+            max: 1
```

Comment:
```suggestion
/*      - !type:SpawnEntitiesBehavior
        spawn:
          CableApcStack1:
            min: 1
            max: 1 TODO CM14 APC cable */ 
```

Reply:
Commented out. Also fixed a bug with windoor deconstruction that I missed. Remember that YAML uses `#` for comments and doesn't support block comments.